power_grid_model_c/power_grid_model/include/power_grid_model/math_solver/iterative_pf_solver.hpp

```diff
@@ -20,7 +20,7 @@
 #include "../three_phase_tensor.hpp"
 #include "../timer.hpp"
 
-namespace power_grid_model::math_model_impl {
+namespace power_grid_model::math_solver {
```

Comment:
optionally:

```suggestion
namespace power_grid_model::math_solver::iterative_pf {
```

with according `using` statement ofcourse (optional because there's no other functions here)